frontend/server/src/main/java/org/pytorch/serve/wlm/BatchAggregator.java

```diff
@@ -61,13 +59,13 @@ public BaseModelRequest getRequest(String threadName, WorkerState state)
         return req;
     }
 
-    public void sendResponse(ModelWorkerResponse message) {
+    public boolean sendResponse(ModelWorkerResponse message) {
```

Comment:
nit: A java doc here might be helpful, specifically highlighting what the return value denotes.